apps/src/templates/manageStudents/ManageStudentsAgeCell.jsx

```diff
@@ -11,10 +11,15 @@ class ManageStudentAgeCell extends Component {
     editedValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     // Provided by redux
     editStudent: PropTypes.func.isRequired,
+    setSharingDefault: PropTypes.func.isRequired,
   };
 
+  // For privacy reasons, we disable sharing by default if the student is under the age of 13 if the age was previously not set.
   onChangeAge = (e) => {
     this.props.editStudent(this.props.id, {age: e.target.value});
+    if (this.props.age === '') {
```

Comment:
does this also need to check if the share column is open? or do we not care - I'm still unclear on this after reading through Poorva's comments.

Reply:
I don't think we care.  My interpretation is that we would want the share setting to update accordingly even if the teacher isn't looking at the share column.